### DISCUSSION TITLE
MIMIC-III subject id matching with physioBank folder and files

### DISCUSSION THREAD

**Discussion by Khaled-Mohammed-Abdelgaber** [MIT-LCP/mimic-code#1496] [MIMIC-General]
Hello team,
I have downloaded MIMIC-III demo files and I can not figure out how to match between subject id and its corresponding waveforms on physioBank. in physioBank divided into folders (from part 1 to part 9), each folder is divided into sub-folders and each subfolder is divided into multiples of records
I hope I will find the answer here.
Thanks in advance.

**Comment by alistairewj** [comment]
Duplicate of #1498
